dd2480/README.md

```diff
@@ -285,3 +294,4 @@ There are many opportunities to split up the code into smaller functions. We wou
 
```

Comment:
Could we also document the outcome of the refactoring for FieldNameLabel? Write down what CCN we started with and the reduction after refactoring

Reply:
I assume you mean RTFChars?
The FieldNameLabel refactor is related to #26.
I added the CCN reduction info in sha [1483301](https://github.com/DD2480-gr25/jabref/pull/30/commits/14833013ecd47586a8daa46fbe7fe3c647fb3690).